### PR TITLE
Tidy up entity header, including proper social icons.

### PR DESCRIPTION
**Pull request by kadams54**

### CODE REVIEW

**Comment by kadams54** [comment]
## QA

See the bugs below.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3877/
Test FAILed.

**Comment by makyo** [comment]
Sweet.  :+1: QA :ok:

**Comment by kadams54** [comment]
Manual CI build was successful: http://ci.jujugui.org:8080/job/juju-gui/3880/

File: jujugui/static/gui/src/app/assets/svgs/icon-social-google.svg [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/assets/svgs/icon-social-google.svg]: I'm assuming these are minified versions of the old icons, but just FYI the new svg sprite system actually does this for us!

File: jujugui/static/gui/src/app/components/entity-header/entity-header.js [hunk 2]
```diff
@@ -187,17 +187,21 @@ YUI.add('entity-header', function() {
...
                     <a id="item-twitter"
                       target="_blank"
-                      href={twitterUrl}
-                      className="entity-header__social-item--twitter">
-                      Share on Twitter
+                      href={twitterUrl}>
+                      <juju.components.SvgIcon
+                        name="icon-social-twitter"
+                        height="35"
+                        width="35"/>
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/entity-header/entity-header.js]: If you want a uniform width/height you can just use `size="35"`

**Comment by huwshimi** [comment]
Looks good @kadams54, one tiny thing where there's about 5px of space below the social icons so the bottom space ends up being about 55px. :+1: with that fixed.

File: jujugui/static/gui/src/app/assets/svgs/icon-social-google.svg (hunk 1 above)
Comment (kadams54) [inline jujugui/static/gui/src/app/assets/svgs/icon-social-google.svg]: They're new icons, though very similar to the old ones.

**Comment by kadams54** [comment]
Thanks all! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3892/
Test PASSed.

**Pull request merged**